statistics-production/ees.qmd

```diff
@@ -893,8 +912,23 @@ The above code should produce something along the lines of the following plots:
 
 ---
 
-## Content
+#### Embedding R-Shiny blocks
+
+---
+
+If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R-Shiny produced chart.
```

Comment:
Is R-Shiny the correct way to write it? We have other variants elsewhere (e.g. "R Shiny", "RShiny")

Reply:
Nope, should be R Shiny! Fixed, and did a sweep of other ones too